src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -122,32 +127,55 @@ export const Browser: FunctionComponent<BrowserProps> = (
     if (props.drawerOpened === false) {
       return;
     }
-
     props.toggleDrawer(false);
   }, [props.drawerOpened]);
 
+  const [trackAnimation, setTrackAnimation] = useSpring(() => ({
+    config: { tension: 280, friction: 45 },
+    height: '100%',
+    width: 'calc(-36px + 100vw )'
+  }));
+
+  const getbrowserWidth = (): string => {
+    if (props.drawerOpened) {
+      return 'calc(41px + 0vw)';
+    }
+    return props.trackPanelOpened
+      ? 'calc(-356px + 100vw)'
+      : 'calc(-36px + 100vw)';
+  };
+
+  useEffect(() => {
+    setTrackAnimation({
+      width: getbrowserWidth()
```

Comment:
Please change `getbrowserWidth` to `getBrowserWidth`.

Reply:
That's done